package.json

```diff
@@ -5,54 +5,29 @@
   "main": "./src/client.js",
   "scripts": {
     "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp",
-    "audit:ci": "audit-ci --low",
     "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
-    "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
-    "build": "rm -rf build && cp envConfig/local.env .env && npm run webpack:prod && npm run test:size",
+    "build": "rm -rf build && cp envConfig/local.env .env && NODE_ENV=production webpack && node ./scripts/bundleSize.js",
+    "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:storybook": "build-storybook -c .storybook -o storybook_dist",
-    "build:test": "cp envConfig/test.env .env && npm run webpack:prod",
-    "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
-    "cypress:proxy": "export NO_PROXY=localhost,localhost.bbc.com && cypress",
-    "cypress": "npm run cypress:proxy -- run",
-    "cypress:interactive": "npm run cypress:proxy -- open",
-    "cypress:3rdParty": "npm run cypress -- --project ./3rdPartyCypress/.",
+    "build:test": "cp envConfig/test.env .env && NODE_ENV=production webpack",
+    "cypress": "export NO_PROXY=localhost,localhost.bbc.com && cypress run",
+    "cypress:interactive": "export NO_PROXY=localhost,localhost.bbc.com && cypress --open",
+    "cypress:3rdParty": "npm run cypress --project ./3rdPartyCypress/.",
     "cypress:3rdParty:interactive": "npm run cypress:interactive -- --project ./3rdPartyCypress/.",
-    "dataValidate": "node dataValidator/index.js",
-    "dataValidate:debug": "npm run dataValidate --DEBUG_MODE=true",
     "dev": "rm -rf build && cp envConfig/local.env .env && run-p webpack:dev:client webpack:dev:server",
-    "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; perl -pi -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "killApp": "(lsof -t -i:7080 | xargs kill)",
-    "killStorybook": "(lsof -t -i:9001 | xargs kill)",
-    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; \n git update-index --assume-unchanged .env",
-    "speculate": "speculate",
```

Comment:
bbc/simorgh-infrastructure#666 still uses `npm run speculate` - you could change the `spec` target in its Makefile to use `npx speculate@2.1.0` instead

Reply:
Updated.